docs/run/quantum-serverless.mdx

```diff
@@ -5,55 +5,79 @@ description: Run workloads with Quantum Serverless
 
 # Run workloads with Quantum Serverless
 
-Premium users can run their workloads remotely on classical compute made available through IBM Quantum Platform.
+Premium users can build, deploy, and run their workloads remotely on classical compute made available through the IBM Quantum Platform.
+
+Try out the tutorials in [IBM Quantum Learning](https://learning.quantum.ibm.com/catalog/tutorials?topics=qiskit-patterns) and explore more of the features of Quantum Serverless in the [documentation](https://qiskit-extensions.github.io/quantum-serverless/).
```

Comment:
Right after the quantum learning link, we should add a parenthetical note along the lines of:

> Try out the tutorials in [IBM Quantum Learning](https://learning.quantum.ibm.com/catalog/tutorials?topics=qiskit-patterns) (note: these are accessible in the Premium Plan once you have logged into your IBM Quantum account) and explore more of the features of Quantum Serverless in the [open-source documentation](https://qiskit-extensions.github.io/quantum-serverless/).